recipe/meta.yaml

```diff
@@ -28,22 +29,17 @@ requirements:
     - pywin32  # [win]
     - traitlets
 
-{% set skip_all = "not test_not_on_path and not test_path_priority" %}
```

Comment:
would be great! it just sometimes, tests don't plan for the "fun" of running in n layers of docker, cross-device links, etc.